doc/sr_integration.rst

```diff
@@ -0,0 +1,211 @@
+*****************************
+Integrating into a Simulation
+*****************************
+
+========
+Overview
+========
+
+This document provides some general guidelines to integrate the SmartRedis client
+into existing simulation codebase. Developers of these simulation codebases will
+need to identify the exact places to add the code, generally only SmartRedis
+calls will need to be added in two places:
+
```

Comment:
Change to: "need to identify the exact places to add the code. Generally, SmartRedis
calls will only need to be added in two places:"